packages/compass-indexes/src/components/create-index-form/create-index-form.tsx

```diff
@@ -86,6 +86,10 @@ function CreateIndexForm({
   const showIndexesGuidanceQueryFlow =
     showIndexesGuidanceVariant && currentTab === 'QueryFlow';
 
+  const dbCollectionNames = namespace.split('.');
+  const dbName = dbCollectionNames[0];
+  const collectionName = dbCollectionNames[1];
```

Comment:
Collection names can have `.` characters in them. So we should parse this a bit differently. In other places in Compass we use the `mongodb-ns` package like so: https://github.com/mongodb-js/compass/blob/f1b976eea70a90c25f55a2ea1c97f1d75dc5d2a1/packages/compass-aggregations/src/stores/store.ts#L120